source/gui/contextHelp.py

```diff
@@ -1,16 +1,16 @@
 # -*- coding: UTF-8 -*-
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2017-2020 NV Access Limited, Thomas Stivers
+# Copyright (C) 2017-2022 NV Access Limited, Thomas Stivers
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 import os
 import tempfile
 import typing
 
+from gui import blockAction
 import wx

```

Comment:
Shouldn't this import of wx appear higher in the import list, per pep-8? If you're changing this section anyway...

Reply:
While our imports are rarely grouped right, I'd rather not introduce a pep-8 violation in this PR. I've fixed this in 3392b93